cmd/syft/cli/cli.go

```diff
@@ -16,38 +17,39 @@ import (
 	"github.com/anchore/syft/internal/redact"
 )
 
-// Application constructs the `syft packages` command, aliases the root command to `syft packages`,
-// and constructs the `syft power-user` command. It is also responsible for
-// organizing flag usage and injecting the application config for each command.
-// It also constructs the syft attest command and the syft version command.
-// `RunE` is the earliest that the complete application configuration can be loaded.
+// Application returns the CLIO application for the syft CLI.
 func Application(id clio.Identification) clio.Application {
-	app, _ := create(id)
+	app, _ := Create(id, os.Stdout)
 	return app
 }
 
 // Command returns the root command for the syft CLI application. This is useful for embedding the entire syft CLI
 // into an existing application.
 func Command(id clio.Identification) *cobra.Command {
-	_, cmd := create(id)
+	_, cmd := Create(id, os.Stdout)
 	return cmd
 }
```

Comment:
`Application` get get removed, I think. Does it make sense to just remove `Command` also, in favor of using the `Create` function?

Reply:
only the command or the application should be used, so exposing `Create` which returns both is confusing (which is why there wasn't a single function for this to begin with). Why not extend the existing `Application` to accept a writer?